### ISSUE TITLE
Slow encoding of video

### ISSUE THREAD

**Issue by hunterhogan** [MambaCodes/BeAnonymous#5]
video_generator.py creates the video using `write_videofile()`, which requires re-encoding the entire video. Furthermore, I _think_ this code is **recursively** concatenating the video clip one clip at a time, which is inefficient, to say the least.
```
#Generating Stock Video 
    if (stockclip.duration < final_audio.duration):
        while (stockclip.duration < final_audio.duration):
            stockclip=concatenate_videoclips([stockclip,stockclip])
```

Since the video files are merely concatenated, it would be much faster to stream copy the files. I don't know how to write the command using `moviepy` but I know the command for FFmpeg. The following command should work for videos _without_ the introduction. The FFmpeg wiki has an excellent explanation of [concatenation](https://trac.ffmpeg.org/wiki/Concatenate). (To make it easier to read, it is split across multiple lines, but it should all be on one line.)

```
ffmpeg
-hide_banner 
-loglevel quiet
-an -stream_loop -1 -t final_audio.duration -i "stockclip"
-i "final_audio"
-codec:video copy 
"FinalPath"
```

**Comment by iamDyeus** [comment]
Yeah, am Aware that the video encoding is slow. When I first started the project, I didn't focus on efficiency. If it worked, I left it. It's great that you understood the code. Lately, I've been rewriting the entire code for better performance. I've also integrated Eleven Labs Speech for better text-to-speech. Honestly, even I found my own initial code a bit tricky to follow!

**Comment by iamDyeus** [comment]
Also, I'll make sure to enhance the video encoding in the upcoming commit, which I plan to make by the end of next month. Thanks for pointing out the FFMPEG command and resources.

**Comment by iamDyeus** [comment]
not gonna lie, completely forgot about this repo; 
no worries will get all the issues sorted this weekend.

**Comment by hunterhogan** [comment]
I forgot, too. I just discovered how to see all of my open Issues in one place, and I was closing stuff I wasn't working on. So, please don't put out any effort for me. :)
